### DISCUSSION TITLE
PDF Generator (Plugin) Wheelchair Symbol is a ? in Table format

### DISCUSSION THREAD

**Discussion by sflwa** [code4recovery/pdf#78] [Q&A] [answered]
I am trying to debug / fix this - I believe the issue is Adobe doesn't like the Unicode &#9855; so it is showing as a ? instead.
I tried to use fontawesome instead - which should work, but need to reference it somewhere in the code.

I also tried it as \xE2\x99\xBF - but that just does the literal output and not the symbol

This is specifically dealing with tabletcpdf.php Line 108

**Comment by SIA-WEBSUP** [comment]
This question should be moved to the PDF repo (https://github.com/code4recovery/pdf/discussions).

**Reply by joshreisner** [reply]
ok i transferred it - i'm not so sure this relates to pdf.code4recovery.org - do we have a `tabletcpdf.php` in the repo? 

@sflwa could you share the steps to reproduce? thanks

**Reply by anchovie91471** [reply]
Thanks @joshreisner. I was about to ask that. That file name doesn't exist in our software.

@sflwa Is it possible you are using a different plugin to generate your pdf? There is an older package in the wild that is an entirely separate plugin.

**Reply by joshreisner** [reply]
yah, is it this one? https://wordpress.org/plugins/12-step-meeting-pdf-generator/

**Reply by sflwa** [reply, marked as answer]
Thank you for your replies - I didn't realize it was something separate - I will as the plugin author over there.  You can close this issue
